packages/schema/lib/functional-constraints/pollingThrottle.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const _ = require('lodash');
+const jsonschema = require('jsonschema');
+
+const pollingThrottle = (definition) => {
+  const errors = [];
+  const actionType = 'triggers';
+
+  if (definition[actionType]) {
+    _.each(definition[actionType], (actionDef) => {
+      if (actionDef.operation && actionDef.operation.throttle && _.has(actionDef.operation.throttle, 'retry') && actionDef.operation.type === 'polling') {
```

Comment:
`polling` is the default, so a polling trigger can have an undefined `operation.type`.

```suggestion
      if (
        actionDef.operation &&
        actionDef.operation.throttle &&
        _.has(actionDef.operation.throttle, 'retry') &&
        (!actionDef.operation.type || actionDef.operation.type === 'polling')
      ) {
```

Reply:
Then should we also have a functional constraint that stops hook triggers from having `throttle`? @kola-er

Reply:
Yeah, probably block throttle entirely for any type of trigger.

It's not completely in-line with https://coda.io/d/Edge-Team_dCaLL2dK8Zu/No-More-Action-Types_suat9 as it should depend on _how_ an action is used, but pragmatically the correct thing to do for now.

Reply:
We can remove the `throttle` entirely from hook trigger, but we still need it for the polling trigger. As pointed out [here](https://zapier.slack.com/archives/C04P5J5R8E4/p1716513877072559), it still has a use-case.

>As we're planning on increasing the polling rate for all integrations, we're concerned about one or two apps which might become rate limited and thus, want to throttle them in advance to avoid issues.

Reply:
Done @eliangcs